src/pro/sky/lesson3/Main.java

```diff
@@ -9,62 +9,95 @@ public static void main(String[] args) {
 
         var box = 16;
         System.out.println(box);
+
         byte eggs = 126;
         System.out.println(eggs);
+
         short socks = 32_766;
         System.out.println(socks);
+
         int noodle = 112_542_4;
         System.out.println(noodle);
+
         long road = 8_925_423L;
         System.out.println(road);
+
         float nuts = 5.56f;
         System.out.println(nuts);
+
         double parts = 5.56674;
         System.out.println(parts);
 
+
         System.out.println("Задание 2 ");
 
         float firstBoxer = 78.2f;
```

Comment:
```suggestion
        float firstBoxerWeight = 78.2f;
```
такое имя переменной более полно отражает её содержимое. То же самое и для другой аналогичной